### ISSUE TITLE
v5.x: improve accessibility

### ISSUE THREAD

**Issue by XhmikosR** [squidfunk/mkdocs-material#1611]
After updating to v5.1.0 I see some new Lighthouse issues, which are real issues

https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocs.pi-hole.net%2F

> Links do not have a discernible name

You are probably missing the `aria-label` on the `a` tags.

**Comment by squidfunk** [comment]
This was already discussed as part of #1488

**Comment by XhmikosR** [comment]
Yeah, title alone is not right. It needs an `aria-label` on the `a` tag and perhaps `aria-hidden` in the SVG itself.

**Comment by squidfunk** [comment]
You can PR the necessary template changes if you like. I've currently no bandwidth to fix and test this. Note, however, that I'm not interested in introducing another parameter for the social links, as discussed in the issue.

**Comment by XhmikosR** [comment]
Sure thing, just leave the issue open until I find some time to work on this.

**Comment by XhmikosR** [comment]
I looked at this and I can't find a simple way to add `aria-hidden` in the SVGs. The only other solution I can think of is adding a new config `text` and if set do something like this:

```html
{% if config.extra.social %}
  <div class="md-footer-social">
    {% for social in config.extra.social %}
      {% set _,rest = social.link.split("//") %}
      {% set domain = rest.split("/")[0] %}
      <a
        href="{{ social.link }}"
        target="_blank" rel="noopener"
        title="{{ domain }}"
        class="md-footer-social__link"
      >
        {% include ".icons/" ~ social.icon ~ ".svg" %}
        {% if social.text %}
        <span class="sr-only">{{ social.text }}<span>
        {% endif %}
      </a>
    {% endfor %}
  </div>
{% endif %}
```

And something similar for the Edit button.

**Comment by squidfunk** [comment]
Yeah, as I said, we won't do that. It overcomplicates things with no upside. The only result would be to silence Lighthouse. Given the issues this project receives, people are already struggling with the configuration options that are available, so I don't want to introduce another one.
